tardis/io/atom_data/collision_data.py

```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
+
+import pandas as pd
+
+
+@dataclass
+class CollisionData:
```

Comment:
I think that is a special CMFGEN collision class. We need to look into this. There are two different formats

Reply:
I think for now, I would have a Chianti1CollisionData and a CMFGENCollisionData